tests/testPhotoCal.py

```diff
@@ -42,6 +43,15 @@
 Log(Log.getDefaultLog(), "meas.astrom.sip", Log.WARN)
 Log(Log.getDefaultLog(), "astrometricSolver", Log.WARN)
 
+testColorterms = ColortermLibrary(data={
+    "test*": ColortermDict(data={
```

Comment:
The colorterm loading allows matching with wildcards, but that functionality doesn't appear to be tested here (`self.config.photoCatName = "test" `) so it should be removed or the test should be changed to use a `photoCatName = "testFooBar"` or similar.